api/src/test/java/uk/gov/dhsc/htbhf/claimant/communications/UpcomingBirthdayEmailHandlerTest.java

```diff
@@ -44,11 +43,14 @@
     private static final LocalDate TURNS_ONE_ON_DAY_OF_NEXT_PAYMENT_CYCLE = START_OF_NEXT_CYCLE.minusYears(1);
     private static final LocalDate TURNS_ONE_IN_FIRST_WEEK_OF_NEXT_PAYMENT_CYCLE = START_OF_NEXT_CYCLE.minusYears(1).plusDays(4);
     private static final Integer NUMBER_OF_CALCULATION_PERIODS = 4;
+    private static final Map<String, Object> COMMON_EMAIL_MAP = Map.of(FIRST_NAME.getTemplateKeyName(), VALID_FIRST_NAME);
```

Comment:
Is it sufficient to have such a small map as this for these test? Might be but thought I'd ask the question.

Reply:
I can't see how the size of the map matters, the key thing is that we test that we have the map and add to it.

Reply:
We're just testing that the contents of this map are included. For that purpose, a single entry is enough